client/src/views/Block-detail.vue

```diff
@@ -61,74 +61,74 @@
   </div>
 </template>
 <script>
-import w1 from "@/components/inforow.vue";
-import w2 from "@/components/Table-dynamic.vue";
+import w1 from '@/components/inforow.vue'
+import w2 from '@/components/Table-dynamic.vue'
 
-import DataService from "../data-service";
-var s = "";
+import DataService from '../data-service'
+var s = ''
 export default {
-  name: "block",
+  name: 'block',
   components: {
     inforow: w1,
     datatable: w2
   },
-  data() {
+  data () {
     return {
       block_id: this.$route.params.blockid,
       blockdetails: {
-        Height: "",
-        Timestamp: "",
-        Difficulty: "",
-        Fees: "",
-        "Total Transactions": "",
-        Harvester: "",
-        "Block Hash": ""
+        Height: '',
+        Timestamp: '',
+        Difficulty: '',
+        Fees: '',
+        'Total Transactions': '',
+        Harvester: '',
+        'Block Hash': ''
       },
       table_data: {
         head: [
-          "#",
-          "Timestamp",
-          "Transaction Type",
-          "Fees",
-          "Signer",
-          "Tx Hash"
+          '#',
+          'Timestamp',
+          'Transaction Type',
+          'Fees',
+          'Signer',
+          'Tx Hash'
         ],
         data: []
       },
-      loading: true,
-    };
+      loading: true
+    }
   },
-  created() {
-    this.asyncData();
+  created () {
+    this.asyncData()
   },
   watch: {
-    $route: "asyncData"
+    $route: 'asyncData'
   },
   methods: {
-    asyncData() {
-      let self = this;
-      this.block_id = this.$route.params.blockid;
-      DataService.getBlockinfo(this.block_id).then(function(data) {
-        self.blockdetails["Height"] = data.blockdata.height;
-        self.blockdetails["Timestamp"] = data.blockdata.date;
-        self.blockdetails["Difficulty"] = data.blockdata.difficulty;
-        self.blockdetails["Fees"] = data.blockdata.totalFee;
-        self.blockdetails["Total Transactions"] =data.blockdata.numTransactions;
-        self.blockdetails["Harvester"] = data.blockdata.signer.address.address;
-        self.blockdetails["Block Hash"] = data.blockdata.blockTransactionsHash;
+    asyncData () {
+      let self = this
+      this.block_id = this.$route.params.blockid
+      DataService.getBlockinfo(this.block_id).then(function (data) {
+        self.blockdetails['Height'] = data.blockdata.height
+        self.blockdetails['Timestamp'] = data.blockdata.date
+        self.blockdetails['Difficulty'] = data.blockdata.difficulty
+        self.blockdetails['Fees'] = data.blockdata.totalFee
+        self.blockdetails['Total Transactions'] = data.blockdata.numTransactions
+        self.blockdetails['Harvester'] = data.blockdata.signer.address.address
+        self.blockdetails['Block Hash'] = data.blockdata.blockTransactionsHash
```

Comment:
wrong data .

it should `data.blockdata.hash`

Reply:
changed